src/views/LandingPage/Sections/SectionLeksehjelp.tsx

```diff
@@ -1,22 +1,84 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Dropdown, { Option } from 'react-dropdown';
+
+//Styles
 import '../../../styles/LandingPage.less';
 
+//Interfaces
+import { ICourse, IStatus } from '../../../interfaces';
+
+//Services
+import { getCourseList, getCourseStatus } from '../../../services/api-service';
+
 const SectionLeksehjelp = () => {
+  const [courses, setCourses] = useState([] as ICourse[]);
+  const [courseStatus, setCourseStatus] = useState([] as IStatus[]);
+  const [formControls, setFormControls] = useState({
```

Comment:
Dropdown har en Option interface det går an å bruke her

Reply:
Brukt interfacet lenger ned, lar den stå som dette for nå.